### DISCUSSION TITLE
Refused to connect to 'https://api.bitwarden.com/accounts/profile' because it violates the document's Content Security Policy.

### DISCUSSION THREAD

**Discussion by florian-sabonchi** [dani-garcia/vaultwarden#3175] [Q&A]
### Subject of the issue

* vaultwarden version:  Version 1.27.0 

* Install method: Docker 

* Clients used:  web

* Reverse proxy and version: treafik

* MySQL/MariaDB or PostgreSQL version: Postgres

* Other relevant details: Version 2022.12.0

### Steps to reproduce

login to vault warden sometimes passwords are empty
### Expected behaviour
passwords should be loaded 

### Actual behaviour
passwords are not loaded-->

**Comment by florian-sabonchi** [comment]
![Screenshot from 2023-01-26 10-22-17](https://user-images.githubusercontent.com/54689374/214801370-7242da7b-bb2c-4009-83c8-51cd0fad7935.png)

**Comment by BlackDex** [comment]
I'm not sure what you are trying to do, but Vaultwarden never ever connects to `api.bitwarden.com` for anything.
There is just one feature which could cause Vaultwarden to do anything on a Bitwarden endpoint, and those are the favicon's if you configure that you want to use there icon fetching endpoint.

Other then that, all should be fully independent from any Bitwarden endpoint.

I suggest to double check your docker/reverse-proxy configuration.

**Comment by florian-sabonchi** [comment]
Hi, Thanks for your feedback it has indeed puzzled me that is trying to send a request to bitwarden. I do not know what I can have configured wrong, I will check my configuration again.
